tensorflow/c/eager/BUILD

```diff
@@ -227,6 +227,43 @@ tf_cuda_cc_test(
     ],
 )
 
+cc_library(
```

Comment:
Could you add `gradients_util.h`, `gradients_util.cc`, `gradient_checker.h`, `gradient_checker.cc` to the `exclude` list of `srcs` filegroup at the end of this file? That should prevent these files from reaching the mobile build which should fix the android tests I believe.